### PR TITLE
Fusion Filtering Staggered PR7

### PR DESCRIPTION
**Pull request by runjin326**
### Purpose/implementation Section
Changes in README and .sh run script

#### What GitHub issue does your pull request address?

https://github.com/PediatricOpenTargets/ticket-tracker/issues/7

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/fusion_filtering/README.md [hunk 1]
```diff
@@ -43,35 +42,39 @@ The code to generate genelistreference.txt and fusionreference.txt is available
...
-Oncogene annotated fusions do not need to be in both callers to be retained however if these fusions are found in more than 4 histologies we treat them as false calls and remove them.
-To scavenge back non-oncogenic fusions that are recurrently found uniquely in a broad_histology we kept fusions that were called by both callers and if >2 samples per histology called the fusion.
-We removed the non-oncogenic fusions with genes fused more than 5 times in a samples or found in more than 1 histology as potential artifact. 
+Oncogene annotated fusions do not need to be in both callers to be retained however if these fusions are found in more than 4 cancer groups we treat them as false calls and remove them.
+To scavenge back non-oncogenic fusions that are recurrently found uniquely in a cancer group we kept fusions that were called by both callers and if >2 samples per hcancer group called the fusion.
```
Comment (kgaonkar6) [inline analyses/fusion_filtering/README.md]:
```suggestion
To scavenge back non-oncogenic fusions that are recurrently found uniquely in a cancer group we kept fusions that were called by both callers and if >2 samples per cancer group called the fusion.
```

File: analyses/fusion_filtering/README.md [hunk 2]
```diff
@@ -43,35 +42,39 @@ The code to generate genelistreference.txt and fusionreference.txt is available
...
+`02-fusion-filtering.R` : Filters artifacts by removing readthroughs and NEIGHBORS from annots column; annots column also contains red flag databases that are used to further filter common/normal occuring fusions; filters all fusions where both genes are expressed TPM < 1 are removed as non-expressed; requires fusions to have at least 1 JunctionSpanningRead and SpanningFragCount-JunctionReadCount to be <100
 
-`03-Calc-zscore-annotate.R` : Calculates z-score for gene fused gene's expression compared to GTeX brain samples and annotates differential expression status
+`03-Calc-zscore-annotate.R` : Calculates z-score for gene fused gene's expression compared to GTeX normal samples. Please note that for different cohort, different normal expression files are used and hence the input for the cohort of interest and the input for the normal GTEx expression file needs to be in order
```
Comment (kgaonkar6) [inline analyses/fusion_filtering/README.md]:
```suggestion
`03-Calc-zscore-annotate.R` : Calculates z-score for gene fused gene's expression compared to GTeX normal samples. Please **note** that for different cohort, different normal expression files are used and hence the input values for the cohort of interest (--cohortInterest) and the input values for the normal GTEx expression file (--normalExpressionMatrix) need to be in the same order.
```

**Review by kgaonkar6 — COMMENTED** [review]
Please fix the conflict to the bash script here, then it can be approved.

**Comment by runjin326** [comment]
@kgaonkar6, sorry about that! All fixed now!

**Review by kgaonkar6 — APPROVED** [review]
👍

**Comment by jharenza** [comment]
I will review this once #40 and #41 are merged, so there aren't so many file changes :)

**Comment by kgaonkar6** [comment]
Last commit merges up-to-date  `dev` to this branch

File: analyses/fusion_filtering/README.md [hunk 3]
```diff
@@ -37,41 +36,45 @@ Annotation | File | Source
...
+Oncogene annotated fusions do not need to be in both callers to be retained however if these fusions are found in more than 4 broad histologies we treat them as false calls and remove them (this filtering step happen in the next step).
+To scavenge back non-oncogenic fusions that are recurrently found uniquely in a cancer group we kept fusions that were called by both callers and if >2 samples per cancer group called the fusion.
+We removed the non-oncogenic fusions with genes fused more than 5 times in a samples or found in more than 1 cancer group as potential artifact. 
 
-`05-QC_putative_onco_fusion_dustribution.Rmd` : Plots fusions found in multiple (more than 4) histologies in scratch/pbta-fusion-putative-oncogenic-preQC.tsv from 04-project-specific-filtering.Rmd and removes fusion calls found in more than 4 histologies as QC filtering.
+`05-QC_putative_onco_fusion_dustribution.Rmd` : Plots fusions found in multiple (more than 4) cancer groups in scratch/fusion-putative-oncogenic-preQC.tsv from 04-project-specific-filtering.Rmd. For final filtering, fusion calls found in more than 4 broad_histology groups are removed as QC filtering.
```
Comment (jharenza) [inline analyses/fusion_filtering/README.md]:
```suggestion
`05-QC_putative_onco_fusion_distribution.Rmd` : Plots fusions found in multiple (more than 4) cancer groups in scratch/fusion-putative-oncogenic-preQC.tsv from 04-project-specific-filtering.Rmd. For final filtering, fusion calls found in more than 4 broad_histology groups are removed as QC filtering.
```

**Review by jharenza — APPROVED** [review]
one typo, but all good otherwise, thanks!

**Pull request merged**